src/main/java/com/rakovets/course/java/core/practice/oop/classesandobjects/task_4/model/Atm.java

```diff
@@ -0,0 +1,83 @@
+package com.rakovets.course.java.core.practice.oop.classesandobjects.task_4.model;
+
+public class Atm {
+    private int numberBanknotes100;
+    private int numberBanknotes50;
+    private int numberBanknotes20;
+    private int amount;
```

Comment:
What is this?

Reply:
Initially I decided to create a field to store state of a variable `amount,` but i forgot that i shouldn't have added new fields according to task specification)) But we can just create a variable `amount` and use it.